src/static/riot/queues/list.tag

```diff
@@ -0,0 +1,282 @@
+            <th width="125px">Created</th>
+            <th width="50px">Public</th>
+            <th class="right aligned" width="150px">Actions</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr each="{ queue in queues }" class="queue-row">
+            <td>{ queue.name }</td>
+            <td>{ queue.owner }</td>
+            <td>{ timeSince(Date.parse(queue.created_when)) } ago</td>
+            <td class="center aligned">
+                <i class="checkmark box icon green" if="{ queue.is_public }"></i>
+            </td>
+            <td class="right aligned">
+                <span class="popup" data-tooltip="{queue.broker_url}" data-position="left center">
+                    <i class="grey icon eye popup-button" if="{ !!queue.broker_url }"></i>
+                </span>
+                <span class="popup" data-tooltip="Copy Broker URL">
+                    <i class="icon copy outline popup-button" if="{ !!queue.broker_url }"
+                       onclick="{ copy_queue_url.bind(this, queue) }"></i>
+                </span>
+                <span class="popup" data-tooltip="Edit Queue">
+                    <i class="blue icon edit popup-button" if="{ queue.is_owner && !!queue.broker_url }" onclick="{ show_modal.bind(this, queue) }"></i>
+                </span>
+                <span class="popup" data-tooltip="Delete Queue">
+                    <i class="red icon trash alternate outline popup-button" if="{ queue.is_owner && !!queue.broker_url }"
+                       onclick="{ delete_queue.bind(this, queue) }"></i>
+                </span>
+            </td>
+        </tr>
+
+        <tr if="{queues.length === 0}">
+            <td class="center aligned" colspan="4">
+                <em>No Queues Yet!</em>
+            </td>
+        </tr>
+        </tbody>
+        <tfoot>
+        <!-------------------------------------
+                  Pagination
+        ------------------------------------->
+        <tr if="{queues.length > 0 && _.get(pagination, 'next')}">
+            <th colspan="6">
+                <div class="ui right floated pagination menu" if="{queues.length > 0}">
+                    <a show="{!!_.get(pagination, 'previous')}" class="icon item" onclick="{previous_page}">
+                        <i class="left chevron icon"></i>
+                    </a>
+                    <div class="item">
+                        <label>{page}</label>
+                    </div>
+                    <a show="{!!_.get(pagination, 'next')}" class="icon item" onclick="{next_page}">
+                        <i class="right chevron icon"></i>
+                    </a>
+                </div>
+            </th>
+        </tr>
+        </tfoot>
+    </table>
+
+    <div class="ui modal" ref="modal">
+        <div class="header">
+            Queue Form
+        </div>
+        <div class="content">
+            <form class="ui form" ref="form">
+                <div class="ui active tab" data-tab="details">
```

Comment:
what is this for? do we have a menu here?

Reply:
No I missed this when I carried over the HTML for the modal from tasks. Pulling it